jest.config.base.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    roots: [
+        "<rootDir>/src",
+        "<rootDir>/tests"
+    ],
+    transform: {
+        "^.+\\.ts$": "ts-jest"
```

Comment:
The `testRegex` additionally looks for `.tsx` extension, we should add that to this regex too so that they are picked up and ran through `ts-jest`